### ISSUE TITLE
Add test for conda/install.py imports

### ISSUE THREAD

**Issue by kalefranz** [conda/conda#2527]
https://github.com/conda/conda/pull/2526#issuecomment-220751869

Since this type of bug was introduced by at least three people independently, I think it would be good to add a test for this. The test could be something along the lines of:

    tmp_dir = tempfile.mkdtemp()
    shutil.copyfile(conda.install.__file__, tmp_dir)
    sys.path = [tmp_dir] # we also need the standard library here
    import install

Basically, put the `install` module into an empty directory, and test if it can be imported by itself.

**Comment by mcg1969** [comment]
Perhaps it would be sufficient to scan for `(from|import) conda`?
